doc/manual/src/release-notes/rl-next.md

```diff
@@ -28,3 +28,5 @@
 - [`nix-store --query`](@docroot@/command-ref/nix-store/query.md) gained a new type of query: `--valid-derivers`. It returns all `.drv` files in the local store that *can be* used to build the output passed in argument.
 This is in contrast to `--deriver`, which returns the single `.drv` file that *was actually* used to build the output passed in argument. In case the output was substituted from a binary cache,
 this `.drv` file may only exist on said binary cache and not locally.
+
+- [Flake references](@docroot@/command-ref/new-cli/nix3-flake.md#flake-references) now support [percent-encoded](https://datatracker.ietf.org/doc/html/rfc3986#section-2.1) paths. [Path-like references](@docroot@/command-ref/new-cli/nix3-flake.md#path-like-syntax) percent-encode non-URI characters automatically.
```

Comment:
To be honest, I don't fully understand what the new behavior is, and I'm worried this documentation is actively misleading.

So okay, we are now allowed to pass Unicode in file references. How is the percent encoding even relevant in that case? If I read that slice of code correctly it's an implementation detail that funnels the URL through from the CLI to the fetcher. We should not document that externally, it's just irrelevant to users.

Is that still correct?

```suggestion
- [Flake references](@docroot@/command-ref/new-cli/nix3-flake.md#flake-references) now support arbitrary unicode characters.
```

Reply:
Yeah makes sense.